third_party/meerkat/Component/mmSH/android/java/src/com/samsung/android/meerkat/MeerkatServerService.java

```diff
@@ -37,15 +37,15 @@
         System.loadLibrary("meerkat_server_lib");
     }
 
-    private static final String TAG = "MeerkatServer";
+    private static final String TAG = "MeerkatServerService";
     private static final int MEERKAT_NOTIFICATION_ID = 100;
     private static final String MEERKAT_CHANNEL_ID = "meekat_server";
     private static final String MEERKAT_CHANNEL_GROUP_ID = "meekat";
     private static final String CASTANETS_PACKAGE_NAME = "org.chromium.chrome";
     private static final String ACTION_NOTIFICATION_CLICKED = "com.samsung.android.meerkat.NOTIFICATION_CLICKED";
 
     private static Context sApplicationContext;
-    private static Thread mMainThread;
+    private Thread mMainThread;
```

Comment:
Why did you remove "static" here?

Reply:
`mMainThread` is a member of each instance of MeerkatServerService, so I remove `static` keyword.